doc/develop/toolchains/arm_toolchain_for_embedded.rst

```diff
@@ -0,0 +1,70 @@
+.. _toolchain_atfe:
+
+Arm Toolchain for Embedded (ATfE)
+#################################
+
+#. LLVM toolchain by ARM
+
+   #. Arm Toolchain for Embedded (ATfE) is a C and C++ toolchain from Arm based
+      on the free and open-source LLVM Compiler Infrastructure and the Picolib C
+      library for baremetal targets.
+
+   #. ATfE is fined-tuned with a particular focus on performance for newer
+      ARM products (post 2024) like 64-bit Arm Architectures (AArch64),
+      or the M-Profile Vector Extension (MVE, a 32-bit Armv8.1-M extension).
+
+#. Installation
+
+   #. Download and install a `Arm toolchain for embedded`_ build for your operating system
+      and extract it on your file system.
+
+   #. :ref:`Set these environment variables <env_vars>`:
+
+      - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``llvm``.
+      - Set :envvar:`LLVM_TOOLCHAIN_PATH` to the toolchain installation
+      directory.
+
+   #. To check that you have set these variables correctly in your current
+      environment, follow these example shell sessions (the
+      :envvar:`LLVM_TOOLCHAIN_PATH` values may be different on your system):
+
+      .. code-block:: console
+
+         # Linux, macOS:
+         $ echo $ZEPHYR_TOOLCHAIN_VARIANT
+         llvm
+         $ echo $LLVM_TOOLCHAIN_PATH
+         /home/you/Downloads/ATfE
+
+         # Windows:
+         > echo %ZEPHYR_TOOLCHAIN_VARIANT%
+         llvm
+         > echo %LLVM_TOOLCHAIN_PATH%
+         C:\ATfE
```

Comment:
Please use group tags, like this:
```
.. tabs::  
   .. group-tab:: Ubuntu

   Text / console block

   .. group-tab:: Windows

   Text / console block

...
```

so we keep the style and look'n'feel of the general docs.
Ref:
https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-the-zephyr-sdk

https://github.com/zephyrproject-rtos/zephyr/blob/cd43e03bbe24feef4d511587f79f8f1277825ca6/doc/develop/getting_started/index.rst?plain=1#L408-L432